src/pages/patterns/tables/basic-formatting.mdx

```diff
@@ -357,6 +421,8 @@ compared against a blank screen or a spinner.
 - The width of the Skeleton should match the width of the intended cell content, while the Skeleton’s height will be 60%
   of the font size
 
+![Loading states](figma:patterns-tables-basic-formatting-loading-states.png)
```

Comment:
Was this meant to replace or amplify the following example? The content feels redundant.

Reply:
you're absolutely right. it's a duplicate, imported this frame without realizing there's a live example. noted!